docs/how-to/namespace-transition.md

```diff
@@ -177,21 +252,13 @@ release from the `knative-gcp` repository. We will update the page with specific
 1. Delete the old BrokerCell since it is no longer useful.
```

Comment:
Maybe replace "useful" with "being used" to clarify that this shouldn't cause a service disruption. Or be explicit that this won't cause a disruption.

Reply:
Done.